### ISSUE TITLE
Solution doesn't build if "build.cmd" is not run manually first

### ISSUE THREAD

**Issue by OkayX6** [fsprojects/VisualFSharpPowerTools#824]
Yo,

TL;DR: could we remove the `paket.exe` line from .gitignore so that we always commit to the last version?

---

A few days ago (so it was some time after the introduction of paket in VFPT), after many weeks without compiling VFPT in my laptop, I noticed that my solution could not be built from Visual Studio.

The build was failing with Paket v0.4.XX (if I remember correctly it's because it couldn't find the right packages).
I had to run "build.cmd" manually (by closing Visual Studio before) so that paket would auto-update to v0.11.XX and download the necessary packages.

Then only, I could build from Visual Studio.
